assets/js/src/public.jsx

```diff
@@ -82,13 +91,16 @@ jQuery(($) => {
       delay = 0;
     }
     setTimeout(() => {
-      formDiv.addClass('active');
-      checkFormContainer(formDiv);
-
-      if (showOverlay) {
-        formDiv.prev('.mailpoet_form_popup_overlay').addClass('active');
-      }
+      doDisplayForm(formDiv, showOverlay);
     }, delay * 1000);
+
+    const exitIntentEnabled = form.data('exit-intent-enabled');
+    if (exitIntentEnabled) {
+      $(document).on('mouseleave', () => {
+        doDisplayForm(formDiv, showOverlay);
+        $(document).off('mouseleave');
```

Comment:
Based on [docs](https://api.jquery.com/off/) this will unbind all `mouseleave` handlers from document. I think we may potentially break some other plugin. I guess we need to pass a handler as second argument here or maybe more elegant solution would be using [event namespaces](https://api.jquery.com/on/#event-names) so that we just rename `mouseleave` to `mouseleave.mailpoet` according to the docs this should work